.travis.yml

```diff
@@ -20,19 +20,18 @@ script:
   - git clone -b develop https://github.com/lirios/fluid
   - cd fluid
   - git submodule update --init
-  - ./scripts/fetch_icons.sh
   - mkdir build
   - cd build
-  - qmake CONFIG+=use_qt_paths ../fluid.pro
-  - make -j4
+  - qmake LIRI_INSTALL_PREFIX=/usr CONFIG+=use_qt_paths ../fluid.pro
```

Comment:
Probably macOS build fails because of this change. Looks like direct changes to /usr/lib are forbidden by a system policy there.